server/data/it/frasi.txt

```diff
@@ -10762,7 +10762,7 @@ Per il 2010.
 Del successore di Vista, ovvero Windows 7.
 Microsoft sta inoltre cercando di allargare la sua quota di mercato per quanto riguarda le applicazioni Internet.
 Da alcuni anni punta in questa prospettiva sul software di messaggistica istantanea MSN Live Messenger.
-Windows Live Messenger si appoggia al portale MSN.com e al servizio di posta gratuito di Hotmail.
+Windows Live Messenger si appoggia al portale MSN e al servizio di posta gratuito di Hotmail.
```

Comment:
La pronuncia di "MSN" sarebbe "Emme esse enne", per questo credo vada meglio "di Microsoft"
(mentre, ad esempio, "MIT" lo si pronuncia (in genere) così come è scritto)